src/psc_bgk_params.txt

```diff
@@ -0,0 +1,29 @@
+// Pretty much anything can go here. The first word on each line is a parameter name, and the second word is the parameter value.
+// Words are separated by whitespace. Any words after the first two of a line are ignored, as are any lines with <2 words.
+// Duplicate parameter names are overridden.
+// For example, "//" will be parsed as a parameter and will have value "For".
+
+# Don't touch
+box_size .03
+
+m_i 1e9
```

Comment:
I think the parameter stuff is something we need to make a plan for. Anyway, this is fine for now.